src/helper/class-ee-site.php

```diff
@@ -491,21 +513,48 @@ protected function update_alias_domains( $args, $assoc_args ) {
 			$this->site_data['alias_domains'] = implode( ',', $final_alias_domains );
 			$is_ssl                           = $this->site_data['site_ssl'] ? true : false;
 			$this->dump_docker_compose_yml( [ 'nohttps' => $is_ssl ] );
+			\EE_DOCKER::docker_compose_up( $this->site_data['site_fs_path'], ['nginx'] );
```

Comment:
@kirtangajjar you need to handle the `docker-compose up` here as well. Not required for dns method for sure. Please check it is required for webroot method.

Reply:
Actually, that's needed so that alias domains are added in virtual hosts